src/app/header/header.component.html

```diff
@@ -99,17 +102,23 @@ <h2 class="header_descr">
           </i>
         </button>
       </button>
-      <a [routerLink]="'./personal-cabinet/config'"><button mat-menu-item>Особиста
-          інформація</button></a>
-
-      <a [routerLink]="'./personal-cabinet/' + (user$ | async).role + '/info'"> <button mat-menu-item>Інформація про
-          {{roles[(user$ | async).role]}}</button></a>
-      <a [routerLink]="'./personal-cabinet/workshops'"><button mat-menu-item>Мої гуртки</button></a>
-      <a [routerLink]="'./personal-cabinet/applications'"><button mat-menu-item>Мої заяви</button></a>
-      <!-- <a [routerLink]="'./personal-cabinet/messages'"><button mat-menu-item>Повідомлення</button></a> -->
-      <a *ngIf="(user$ | async).role === Role.parent" [routerLink]="'./personal-cabinet/parent/favorite'"><button
-          mat-menu-item>Улюблене</button></a>
-      <button mat-menu-item (click)="logout()">Вийти</button>
+      <ng-container *ngIf="((user$ | async)?.role !== Role.techAdmin)">
```

Comment:
I think it is better to do case-switching in this way: 
```suggestion
    <div *ngIf="((user$ | async)?.role !== Role.techAdmin) then UserView; else AdminView"></div>
    <ng-template #UserView></ng-template >
    <ng-template #AdminView></ng-template >
```

In this case, we check the role only once, in your case the role checking if happening twice, for techadmin and for not techadmin